common/pom.xml

```diff
@@ -18,6 +18,18 @@
         <jackson.modules.version>2.8.5</jackson.modules.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```

Comment:
This will enable the microservices to use the spring cloud bus and receive notifications of configuration changes from the cloud config server.